bintrace.c

```diff
@@ -74,6 +84,10 @@ struct process_t
 	void *base_load_addr;
 };
 
+static void reset_color() { printf("\033[0m"); }
+static void set_red_color() { printf("\033[0;31m"); }
+static void set_green_color() { printf("\033[0;33m"); }
```

Comment:
How about we add string macros for those? Like:
```cpp
#define COLOR_RED "\033[0;31m"
#define COLOR_GREEN "\033[0;33m"
#define COLOR_RESET "\033[0m"

#define STR_RED(s) COLOR_RED s COLOR_RESET
#define STR_GREEN(s) COLOR_GREEN s COLOR_RESET
```
So it can be used like that:
```cpp
printf(COLOR_RED "This text is red" COLOR_RESET " no longer red\n");
printf(STR_GREEN("This text is green") "no longer red\n");
```